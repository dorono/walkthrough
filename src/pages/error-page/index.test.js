import React from 'react';
import {shallow} from 'enzyme';
import ErrorPage from './index';

describe('ErrorPage', () => {
    it('should render without errors', () => {
        shallow(<ErrorPage />);
    });

    it('should match snapshot', () => {
        expect(shallow(<ErrorPage />)).toMatchSnapshot();
    });
});
