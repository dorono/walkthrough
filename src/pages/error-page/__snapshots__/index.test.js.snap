// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorPage should match snapshot 1`] = `
<Route>
  <Component />
</Route>
`;
